### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by krasi-georgiev**
just some housekeeping
no functional changes

would appreciate a fast review to avoid conflict with other PRs

### CODE REVIEW

File: discovery/kubernetes/endpoints.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func NewEndpoints(l log.Logger, svc, eps, pod cache.SharedInformer) *Endpoints {
 	return ep
 }
 
-// Run implements the retrieval.TargetProvider interface.
+// Run implements the discovery.TargetProvider interface.
```
Comment (simonpasquier) [inline discovery/kubernetes/endpoints.go]: The interface is now `discovery.Discoverer` after your refactoring so comments have to be changed accordingly for all SD providers.

Comment (krasi-georgiev) [reply discovery/kubernetes/endpoints.go]: thanks

Comment (cstyan) [reply discovery/kubernetes/endpoints.go]: if you rebase off master, I just fixed this yesterday or the day before :)

**Comment by krasi-georgiev** [comment]
broken down into 2 smaller PRs
https://github.com/prometheus/prometheus/pull/3766
https://github.com/prometheus/prometheus/pull/3779

**Pull request closed without merging**